### PR TITLE
feat: added OTLP export

### PR DESCRIPTION
**Pull request by adubovik**
* Enable OLTP export if `OTEL_EXPORTER_OTLP_TRACES_ENDPOINT` env var is set
* Bumped DIAL SDK version

### CODE REVIEW

**Review by Oleksii-Klimov — DISMISSED** [review]

File: aidial_adapter_bedrock/app.py [hunk 1]
```diff
@@ -16,14 +17,20 @@
...
-
-@app.get("/healthcheck")
-def healthcheck():
-    return fastapi.Response("OK")
+app = DIALApp(
+    description="AWS Bedrock adapter for DIAL API",
+    add_healthcheck=True,
+    telemetry_config=TelemetryConfig(
+        service_name="bedrock",
+        tracing=TracingConfig(oltp_export=OTLP_EXPORT_ENABLED),
```
Comment (Oleksii-Klimov) [inline aidial_adapter_bedrock/app.py]: Looks like the name in the dial sdk should be fixed.

**Comment by adubovik** [comment]
/deploy-review
>Deployment status: success

**Comment by adubovik** [comment]
/deploy-review
>Deployment status: success

**Comment by adubovik** [comment]
/deploy-review
>Deployment status: success

File: Dockerfile [hunk 2]
```diff
@@ -16,6 +17,7 @@ RUN poetry install --no-interaction --no-ansi --no-cache --only main
 FROM python:3.11-alpine as server
 
 RUN apk update && apk upgrade --no-cache libcrypto3 libssl3
+RUN apk add --no-cache alpine-sdk linux-headers
```
Comment (Oleksii-Klimov) [inline Dockerfile]: Once the binaries are built these should no longer be needed.

File: aidial_adapter_bedrock/app.py (hunk 1 above)
Comment (adubovik) [reply aidial_adapter_bedrock/app.py]: Fixed

File: Dockerfile (hunk 2 above)
Comment (adubovik) [reply Dockerfile]: Fixed

**Review by Oleksii-Klimov — APPROVED** [review]

**Pull request merged**